cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -390,12 +441,22 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
     String gcpCmekKeyName = getString(properties, "encryptionKeyName");
     String gcpCmekBucket = getString(properties, "gcsBucket");
 
+    String clusterMetaData = getString(properties, "clusterMetaData");
+    String networkTags = getString(properties, "networkTags");
+    String initActions = getString(properties, "initActions");
+    boolean autoScaling = Boolean.parseBoolean(properties.getOrDefault("autoScaling",
+                                                                             "false"));
```

Comment:
You don't need `getOrDefault`. Just call `get`. The `Boolean.parseBoolean` will return false if the given value is `null`.

Reply:
Done